util/common.go

```diff
@@ -28,6 +28,16 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+type WorkerSliceGatewayNetworkAddresses struct {
```

Comment:
Why did you move WorkerSliceGatewayNetworkAddresses to utils/common This doesn't seem to best practice

Reply:
Yeah it has to be moved back!

Reply:
Due to import cycle issues.
service package imports mock package.
mock package imports service package.

Hence it was moved here!